server/src/storage/compressed_data_manager.rs

```diff
@@ -54,6 +55,11 @@ pub(super) struct CompressedDataManager {
     compressed_remaining_memory_in_bytes: isize,
     /// Reference to the schema for compressed data buffers.
     compressed_schema: CompressedSchema,
+    /// Log of the used compressed memory in bytes, updated every time the used memory changes.
+    pub(super) used_compressed_memory_log: Log,
+    /// Log of the total used disk space in bytes, updated every time a new compressed file is
+    /// saved to disk.
+    pub used_disk_space_log: Arc<RwLock<Log>>,
```

Comment:
I have added a comment to the place where the `used_disk_space_metric` is initialized that explains it to avoid having to explain it in each struct field.